### PR TITLE
add steps to remove liquidity with Etherscan

### PR DESCRIPTION
**Pull request by kremalicious**
In a new `docs/` folder.

### CODE REVIEW

File: docs/remove-liquidity-v4-etherscan.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Remove liquidity from deprecated Ocean Market v4 pools via Etherscan
+
+## Get your balance of pool share tokens
+
+1. Go to the pool's Etherscan/Polygonscan page. To find it without UI, see through your latest transactions and look for Ocean Pool Token (OPT) transfers. Those transactions always come from the pool contract.
```
[the remark this answers is not in the document]
Comment (mihaisc) [reply docs/remove-liquidity-v4-etherscan.md:5]: Also ignore the transactions from `0x651e859510A219f6CAd57E23926CdE6B9eaee822` , this is the ss bot

File: docs/remove-liquidity-v4-etherscan.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+2. On the pool contract page, go to _Contract_ -> _Read Contract_.
+3. Go to field `20. balanceOf` and insert your ETH address. This will retrieve your pool share token balance.
+4. To convert to wei, go to [eth-converter.com](https://eth-converter.com) and input the retrieved pool shares number in the _Ether_ field
```
Comment (mihaisc) [inline docs/remove-liquidity-v4-etherscan.md:8]: don't really need this step. you can just click on the value and it redirects you to https://rinkeby.etherscan.io/unitconverter?wei=9017293058506014546
image

Comment (mihaisc) [reply docs/remove-liquidity-v4-etherscan.md:8]: And also, this is already in wei, so don't need to convert.

File: docs/remove-liquidity-v4-etherscan.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+1. Go to the pool's Etherscan/Polygonscan page.
+2. Go to _Contract_ -> _Read Contract_.
+3. Go to field `56. totalSupply` to get the total amopunt of pool shares, in wei.
+4. Divide the number by 2 to get the maximum of pool shares you can send in one pool exit transaction. If your number retrieved in former step is bigger, you have to send multiple transactions.
+
+## Exit pool
+
+1. Go to the pool's Etherscan/Polygonscan page.
+2. Go to _Contract_ -> _Write Contract_ and connect your wallet. Be sure to have your wallet connected to network of the pool.
```
Comment (mihaisc) [inline docs/remove-liquidity-v4-etherscan.md:21]: and go to `5. exitswapPoolAmountIn`

**Comment by mihaisc** [comment]
https://rinkeby.etherscan.io/tx/0x5c1dd07d2708573b0ac68b9b6bb4d02c74d4f8aaa99be4c6d0c6c0279d3796ef 🎉

File: docs/remove-liquidity-v4-etherscan.md (hunk 2 above)
Comment (kremalicious) [reply docs/remove-liquidity-v4-etherscan.md]: excellent, good find, much better

**Comment by kremalicious** [comment]
closing in favor of https://github.com/oceanprotocol/docs/pull/1081, where discussion can be continued

**Pull request closed without merging**